### ISSUE TITLE
`MultipartUploadPart.copy_from` is missing dict type for `CopySource` parameter

### ISSUE THREAD

**Issue by mtvx** [youtype/mypy_boto3_builder#169]
**Describe the bug**

In [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.MultipartUploadPart.copy_from) the `MultipartUploadPart.copy_from` supports a union of string and dict:

```py
    CopySource='string' or {'Bucket': 'string', 'Key': 'string', 'VersionId': 'string'},
```

However in the latest type stub there's only `str`:

```py
        CopySource: str,
```

**Additional context**

Happens at least on (the latest) `mypy-boto3-s3==1.24.36.post1`.

Thanks for the great library, these stubs are _essential_ (shame on AWS)!

**Comment by vemel** [comment]
Thank you for the report! I am going to release a fix today.

**Comment by vemel** [comment]
Fixed in [mypy-boto3-s3 1.24.76](https://pypi.org/project/mypy-boto3-s3/).

Please let me know if it works as it should.

**Comment by mtvx** [comment]
Yes, it's *perfect* now. Thank you so much!
